shlr/bochs/src/libbochs.c

```diff
@@ -104,42 +124,47 @@ void bochs_send_cmd(libbochs_t* b, const char * comando, bool bWait) {
 #if __WINDOWS__
 	//lprintf("Enviando comando: %s\n",comando);
 	DWORD dwWritten;
-	bochs_reset_buffer(b);
-	ZeroMemory(cmdBuff,128);
-	sizeSend=sprintf(cmdBuff,"%s\n",comando);
+	bochs_reset_buffer (b);
+	ZeroMemory (cmdBuff,128);
+	sizeSend = sprintf (cmdBuff,"%s\n",comando);
```

Comment:
return value unused. you are recomputing the length again after it. which is unnecessary